Proteomics/tutorials/labelfree_vs_labelled.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: tutorial_hands_on
```

Comment:
Ah, sorry. It is of course mentioned in the `Contributing.md`! I will change to `tutorial`.